google-youtube.html

```diff
@@ -773,7 +783,7 @@
       },
 
       _updatePlaybackStats: function() {
-        this.currenttime = Math.round(this._player.getCurrentTime());
+        this.currenttime = this._player.getCurrentTime();
```

Comment:
Why remove the rounding? This will turn `currenttime` from an "int" to a float with quite a lot of trailing decimal values.

Reply:
Yes, which seems natural to me. _player.getCurrentTime() returns float, how come currenttime loses precision when I didn't ask for it?

But the main reason is because leaving Math.round makes statsUpdateInterval essentially useless.

Reply:
I was under the impression that the default interval was under one second which would cause data-binding thrashing.

Reply:
Regardless, it is a breaking change, can you please hide this unless the value is defined? e.g.

Reply:
Regardless, it is a breaking change, can you please hide this unless the value is defined? e.g.

```js

Reply:
Regardless, it is a breaking change, can you please hide the non-rounding under a boolean flag?